src/library/scala/collection/mutable/Builder.scala

```diff
@@ -164,7 +177,8 @@ class StringBuilder(private val sb: java.lang.StringBuilder) extends AbstractSeq
     *  @param  xs  the characters to be appended.
     *  @return     this StringBuilder.
     */
-  def appendAll(xs: IterableOnce[Char]): StringBuilder = appendAll(ArrayBuffer.from(xs).toArray)
+  def appendAll(xs: IterableOnce[Char]): StringBuilder =
+    appendAll (new ArrayBuilder.ofChar).addAll(xs).result
```

Comment:
```
[error] /home/jenkins/workspace/scala-2.13.x-validate-main/src/library/scala/collection/mutable/Builder.scala:181: overloaded method value appendAll with alternatives:
[error]   (xs: Array[Char])StringBuilder <and>
[error]   (xs: scala.collection.IterableOnce[Char])StringBuilder <and>
[error]   (xs: String)StringBuilder
[error]  cannot be applied to (scala.collection.mutable.ArrayBuilder.ofChar)
[error]     appendAll (new ArrayBuilder.ofChar).addAll(xs).result
[error]     ^
[error] one error found
```

Reply:
it's calling `appendAll(new ArrayBuilder.ofChar)` ugh.

thanks for the heads up